### PR TITLE
Update README, environment, and model file

### PR DESCRIPTION
**Pull request by danfke**
After this pull request, the model creation step should be done.

### CODE REVIEW

**Review by iamMoid — APPROVED** [review]
Champ! Nicely done.

File: README.md [hunk 1]
```diff
@@ -18,12 +18,16 @@ To replicate this analysis, clone this GitHub repository, install the listed [de
...
 # download data
+python src/download_data.py --url="https://opendatatc.blob.core.windows.net/opendatatc/NCDB_2017.csv" --filepath=data/raw/NCDB_2017.csv
 
-#run eda report
+# clean and split data
+python src/clean_split_data.py --input=data/raw/NCDB_2017.csv --output=data/processed/
 
-#clean and split data
+# run eda report
+python src/eda.py --train=data/processed/NCDB_2017.csv --out_dir=results/figures/
```
Comment (gn385x) [inline README.md]: Does NCDB_2017 in processed include train set only?

Comment (gn385x) [reply README.md]: Wait, I just see below that we have train.csv as well. Should it be train.csv here as well?

Comment (danfke) [reply README.md]: Good catch I'll change that to train.csv

File: src/model.py [hunk 2]
```diff
@@ -18,90 +18,53 @@
...
+
     # Get data
-    train_df = pd.read_csv(f"{input}train.csv", low_memory=False).set_index("index").rename_axis(None)
-    X_train = train_df.drop(columns=["FATALITY", "P_ISEV"])
+    train_df = (
+        pd.read_csv(f"{input}", low_memory=False)
+        .set_index("index")
+        .rename_axis(None)
+    )
+    X_train = train_df.drop(columns=["FATALITY"])
```
Comment (SiqiTao) [inline src/model.py:42]: do we still need to remove the `P_ISEV` column?

**Review by gn385x — APPROVED** [review]

File: src/model.py (hunk 2 above)
Comment (SiqiTao) [reply src/model.py:42]: oh nvm i just saw your update in the data cleaning script

**Pull request merged**